wix/Build.OpenJDK_generic.cmd

```diff
@@ -70,6 +70,24 @@ SET PRODUCT_SKU=OpenJDK
 SET PRODUCT_VERSION=%PRODUCT_MAJOR_VERSION%.%PRODUCT_MINOR_VERSION%.%PRODUCT_MAINTENANCE_VERSION%.%PRODUCT_PATCH_VERSION%
 SET ICEDTEAWEB_DIR=.\SourceDir\icedtea-web-image
 
+REM JEP322 alike
+SET JEP322_BASE_NAME="unknown"
+IF %PRODUCT_CATEGORY% EQU jre SET JRE=-jre
+IF %PRODUCT_MAJOR_VERSION% GEQ 10 (
+    IF %PRODUCT_MAJOR_VERSION% EQU 11 SET LTS_VERSION=-LTS
```

Comment:
Maybe make 11 in an array and add 17?

Reply:
>is this trying to solve the problem of creating installers for the jdk tip version (which is currently 15)?

yes

yes we can add 17 for the LTS flag but I have not seen any artifact follow this JEP322 about LTS flag:

>jdk-11.0.7+10/bin/java -version
openjdk version "11.0.7" 2020-04-14
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.7+10)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.7+10, mixed mode)

LTS flag is assigned to JEP322_NAME var
but as we don't follow JEP322 actually this is just for future need if we want to align to JEP322
This code is useless now 

Did you prefer I remove or comment this code ?

Is there any plan to follow JEP322 ?

And I have not seen any documentation/annonce about 17 will be effectively LTS :) .. roadmap can change again..

Reply:
Let's not worry about adding 17 yet, we can do that later